### DISCUSSION TITLE
Error messages in matlab

### DISCUSSION THREAD

**Discussion by logang42** [lpollonini/qt-nirs#10] [General]
Hi, 

When starting up the qtnirs GUI I'm prompted with errors in matlab (screenshot attached). Could someone please explain why this may be happening and any potential solutions. 

note - I'm using matlab 2024a.

![Screenshot 2024-07-18 115304](https://github.com/user-attachments/assets/041fa7c6-c586-401c-83c1-80bf2de0a8bd)

Thanks, 
Logan

**Comment by JakubSchwenkbeck** [comment]
Hey, did you possible forget to download the FileExchange Extension from matlab ( https://de.mathworks.com/matlabcentral/fileexchange/32226-recursive-directory-listing-enhanced-rdir ) to enable the usage of rdir? 
If you have this download, check filepaths which are added in matlab :)
Hope this helps, Jakub

**Comment by smonterohdz** [comment]
Hi Logan,

It seems you did not add the 'utils' folder to MATLAB's path. The 'utils' folder contains a copy of the rdir script. Please verify the QTNIRS folder and its subfolders are added to MATLAB's path. Let me know the results.
Cheers,
Samuel

**Reply by logang42** [reply]
Hi Samuel, 

Ahh yes that worked, thank you. Easy fix that got me one step closer. 

Unfortunately, I'm still encountering issues (as you could probably tell I'm not too proficient with coding). Do you have any solutions for these error messages? Thanks in advanced. 

Cheers, 
Logan 
![Screenshot 2024-07-23 152307](https://github.com/user-attachments/assets/de647296-0295-467d-8c90-e8843a7579d3)

**Comment by smonterohdz** [comment]
Hi Logan
No worries!
Maybe it is something related to the values of the band-pass filtering. Could you please post the values you are using for the different parameters in QTNIRS?

**Reply by logang42** [reply]
Hi Samuel, 

I got it work with a different data set. It appears that the issue with the previous data set was a result of some 'NaN' and 'Inf' data points.  

Thank you for your help.

Cheers,
Logan
